### PR TITLE
divide by total combined doses for %

### PR DESCRIPTION
**Pull request by aaronhans**

### CODE REVIEW

**Review by jbum — APPROVED** [review]
The ratios add up this way. I have no idea what the intention was. :-)

**Comment by aaronhans** [comment]
I am going to merge this to preprod, we'll wait for approval from Angie or Hannah before doing anything to production but based on the chart caption this seems like the right calculation

**Pull request merged**
